test/run-php-linter.sh

```diff
@@ -49,8 +49,6 @@ declare -a tools_list=(
 
 # And on all PHP files in this array
 declare -a test_list=(
-    'integrationtests/LorisIntegrationTest.class.inc'
-    'integrationtests/LorisIntegrationTestWithCandidate.class.inc'
```

Comment:
Are these supposed to be removed?

Reply:
Yes, "test/integrationtests/" folder is integrated L59.

Reply:
This was removed in PR #6800